lib/rails-patterns.rb

```diff
@@ -4,3 +4,6 @@
 require "patterns/collection"
 require "patterns/calculation"
 require "patterns/form"
+require "patterns/rule"
+require "patterns/ruleset"
+require "patterns/strong_ruleset.rb"
```

Comment:
Is this ".rb" needed? If not, I'd remove it to keep consistency.

Yeah, Captain Nitpick here